### ISSUE TITLE
Connect to server - text input turns red after protocol prefix

### ISSUE THREAD

**Issue by Ethanol6** [qarmin/czkawka#872]
Flatpak - 1.12.2
czkawka - 5.0.2

When trying to connect to a server to add a network location the text input for the server address turns red as soon as the protocol prefix is typed. If I click the `?` icon in the text field examples of protocol prefixes are shown. However when actually typing in the address `smb://` (exactly as shown by the examples) the address box turns red when the colon is added. As such I cannot click the "connect" button.

**Comment by qarmin** [comment]
I don't think that czkawka supports smb or other network protocols.
Looks that problem is with used file manager, not Czkawka

**Comment by Ethanol6** [comment]
Its an option in Czkawka when choosing a folder to include in searches.

**Comment by qarmin** [comment]
Nautilus is just one of many file manager that can be used with Czkawka and ability to choose certain paths from it doesn't mean that this will be supported in Czkawka.

To support ssh/smb, good rust wrappers around this libraries are needed(only found crate for ssh - https://crates.io/crates/ssh2) and I would need to rewrite quite big part of code which scan/read metadata or delete files

**Comment by Ethanol6** [comment]
Ok, understood. I use Nemo on my system but the Flatpak uses Nautilus and I did change the Flatpak permissions to allow networking and tried to specifically allow permissions for the remote folder to the Flatpak as well. 

Anyways, I got around this problem by using SSHFS to mount the remote location to a local path I know the Flatpak can find and then it's accessible like a local folder to the Flatpak and Czkawka finds it.
